src/main/java/com/minecolonies/items/ItemAchievementProxy.java

```diff
@@ -0,0 +1,27 @@
+package com.minecolonies.items;
+
+
+/**
+ * This is a proxy item class for achievements.
+ *
+ * Achievements can't have custom icons except using an item or block as proxy.
+ */
+public class ItemAchievementProxy extends AbstractItemMinecolonies
```

Comment:
Currently, through the creative tab only. I will change the pattern and they will be available **only** through NEI/JEI/TMI cheattools or the `/give` command